frontend/index.d.ts

```diff
@@ -95,6 +95,26 @@ interface CAPIType {
     subMetaSectionLinks: SimpleLinkType[],
     subMetaKeywordLinks: SimpleLinkType[],
 }
+
+interface AMPType {
```

Comment:
What are you thoughts on the naming here and also location? As is feels like we could improve both without too much effort.

One option would be to define this type in the Article file and call it ArticleModel or something (i.e. without the AMP prefix and a name a bit more tied to its purpose). And then export it. This would help by:

* not making it a global type (which is probably not a concern for most things, but it does seem good to avoid leaking AMP types to non-AMP stuff)
* gives it a better name (AMPType doesn't really tell you anything - mind you CAPIType is a pretty rubbishy name to start with so we have precedence there!)